lib/navbar/view.dart

```diff
@@ -0,0 +1,42 @@
+import 'dart:html' as html;
+import 'controller.dart';
+
+const NAV_BRAND_ID = 'nav-brand';
+const NAV_LINKS_WRAPPER_ID = 'nav-links-wrapper';
+const NAV_ITEM_CSS_CLASSNAME = 'nav-item';
+const ACTIVE_CSS_CLASSNAME = 'active';
+
+html.SpanElement get _navBrand => html.querySelector('nav #${NAV_BRAND_ID}');
+html.UListElement get _navLinksWrapper =>
+    html.querySelector('nav #${NAV_LINKS_WRAPPER_ID}');
+List<html.LIElement> get _navLinks => html.querySelectorAll(
+    'nav #${NAV_LINKS_WRAPPER_ID} .${NAV_ITEM_CSS_CLASSNAME}');
+
+class View {
+  Controller controller;
+
+  View(this.controller);
+
+  void setNavBrand(String text) {
+    _navBrand.innerText = text;
+  }
+
+  void appendNavLink(String pathname, String label, bool isSelected) {
+    var li = html.LIElement()
+      ..classes = [
+        NAV_ITEM_CSS_CLASSNAME,
+        isSelected ? ACTIVE_CSS_CLASSNAME : '_'
```

Comment:
Something I learnt recently for optionally adding an element to a list when you don't need else clause:
```suggestion
        if (isSelected) ACTIVE_CSS_CLASSNAME
```

Reply:
Ah this is nice! Didn't know this existed - makes me like dart even more :)